modules/tensor_mechanics/src/materials/ComputeMultipleInelasticStress.C

```diff
@@ -75,6 +75,10 @@ ComputeMultipleInelasticStress::ComputeMultipleInelasticStress(const InputParame
     _elasticity_tensor(getMaterialPropertyByName<RankFourTensor>(_base_name + "elasticity_tensor")),
     _elastic_strain_old(getMaterialPropertyOld<RankTwoTensor>(_base_name + "elastic_strain")),
     _strain_increment(getMaterialProperty<RankTwoTensor>(_base_name + "strain_increment")),
+    _has_compliance(hasMaterialProperty<RankFourTensor>(_base_name + "compliance_tensor")),
```

Comment:
Oh yeah, we had this same conversation a few days ago.  I think I've got that in my head now.